src/bundle.rs

```diff
@@ -35,7 +35,10 @@ impl Bundle {
             .into_dictionary()
             .ok_or_else(|| "plist root value is not a dictionary".to_string())?;
 
-        let bundle_name = plist["CFBundleName"].as_string().unwrap();
+        let mut bundle_name = "";
+        if plist.contains_key("CFBundleName") {
+            bundle_name = plist["CFBundleName"].as_string().unwrap();
+        }
```

Comment:
This can be condensed to:

```rust
let bundle_name = plist.get("CFBundleName").map_or("", |s| s.as_string().unwrap());
```

But I don't think using an empty string is a good solution, we should use the filename the bundle already has in that case.